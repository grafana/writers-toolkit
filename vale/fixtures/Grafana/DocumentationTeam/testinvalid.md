Don't use:

- docs squad
- Docs squad
- docs Squad
- Docs Squad
- docs team
- Docs team
- docs Team
- Docs team
