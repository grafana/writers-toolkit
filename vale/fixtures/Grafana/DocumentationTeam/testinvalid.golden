testinvalid.md:3:3:Grafana.DocumentationTeam:Use 'the Grafana Labs documentation team' rather than 'docs squad'.
testinvalid.md:4:3:Grafana.DocumentationTeam:Use 'the Grafana Labs documentation team' rather than 'Docs squad'.
testinvalid.md:5:3:Grafana.DocumentationTeam:Use 'the Grafana Labs documentation team' rather than 'docs Squad'.
testinvalid.md:6:3:Grafana.DocumentationTeam:Use 'the Grafana Labs documentation team' rather than 'Docs Squad'.
testinvalid.md:7:3:Grafana.DocumentationTeam:Use 'the Grafana Labs documentation team' rather than 'docs team'.
testinvalid.md:8:3:Grafana.DocumentationTeam:Use 'the Grafana Labs documentation team' rather than 'Docs team'.
testinvalid.md:9:3:Grafana.DocumentationTeam:Use 'the Grafana Labs documentation team' rather than 'docs Team'.
testinvalid.md:10:3:Grafana.DocumentationTeam:Use 'the Grafana Labs documentation team' rather than 'Docs team'.
